File: src/main/java/item/Item.java
package item;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Positive;


@Data
    @AllArgsConstructor
    @NoArgsConstructor
    @Builder
    public class Item {
        private String itemName;
        @NotNull
        @Positive
        private int quantity;
        @NotNull
        @Positive
        private int price;

    }

File: src/main/kotlin/item/Item.kt
package item

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor
import org.jetbrains.annotations.NotNull
import javax.validation.constraints.Positive


data class Item (
    var itemName: String? = null,
    @field:NotNull @field:Positive
    var quantity: Int = 0,
    @field:NotNull @field:Positive
    var price: Int = 0,
)